icons/calendar-single.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M 11 14 L 12 14" />
+  <path d="M 12 14 L 12 18" />
+  <path d="M16 2v4" />
+  <path d="M3 10h18" />
+  <path d="M8 2v4" />
+  <rect x="3" y="4" width="18" height="18" rx="2" />
```

Comment:
I like the idea, but this can be further optimized:

```suggestion
  <path d="M11 14h1v4" />
  <path d="M16 2v4" />
  <path d="M3 10h18" />
  <path d="M8 2v4" />
  <rect x="3" y="4" width="18" height="18" rx="2" />
```

It should probably also be called `calendar-1` or `calendar-one`, since that's what's depicted.

Reply:
Good one one the name. Probably `calendar-1` seeing `repeat-1`, `volume-1`, `tally-1` etc.

Reply:
Neat optimization for the 1, I've applied it.  
Not sure why you also commented on the other lines, they don't appear changed.

Reply:
> Neat optimization for the 1, I've applied it. Not sure why you also commented on the other lines, they don't appear changed.

Naah, sorry, I was just too lazy to check which lines had actually been changed by the optimisation, it was easier to copy and paste the entire thing. 😅